Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs

```diff
@@ -227,7 +227,11 @@ public float pollingFrequency
         public double currentTimeOffsetToRealtimeSinceStartup => NativeInputSystem.currentTimeOffsetToRealtimeSinceStartup;
         public float unscaledGameTime => Time.unscaledTime;
 
-        public bool runInBackground => Application.runInBackground;
+        public bool runInBackground => Application.runInBackground ||
+        // certain platforms ignore the runInBackground flag and always run. Make sure we're
+        // not running on one of those.
+        // TODO: Add more platforms here as they're discovered.
+        Application.platform == RuntimePlatform.PS5;
```

Comment:
I believe this change is ok to solve this problem, but if ignoring runInBackground is optional I think a more robust solution would have been to have a capability exposed by native stating if the platform respects/supports runInBackground settings and adapt to that.